install_config/persistent_storage/topics/glusterfs_example_full.adoc

```diff
@@ -3,20 +3,37 @@
 ----
 [OSEv3:vars]
 ...
-openshift_registry_selector="role=infra" <1>
+openshift_registry_selector="role=infra"                  <1>
 openshift_hosted_registry_storage_kind=glusterfs
 
+openshift_metrics_hawkular_nodeselector={"role":"infra"}  <1>
+openshift_metrics_cassandra_nodeselector={"role":"infra"} <1>
+openshift_metrics_heapster_nodeselector={"role":"infra"}  <1>
+openshift_metrics_storage_kind=dynamic
+openshift_metrics_cassanda_pvc_storage_class_name="glusterfs-registry-block" <2>
+
+openshift_logging_es_nodeselector={"role":"infra"}        <1>
+openshift_logging_kibana_nodeselector={"role":"infra"}    <1>
+openshift_logging_curator_nodeselector={"role":"infra"}   <1>
+openshift_logging_storage_kind=dynamic
+openshift_logging_es_pvc_size=10Gi                        <3>
+openshift_logging_es_pvc_storage_class_name="glusterfs-registry-block"       <2>
+
 openshift_storage_glusterfs_block_deploy=false
 
 openshift_storage_glusterfs_registry_block_deploy=true
 openshift_storage_glusterfs_registry_block_storageclass=true
-openshift_storage_glusterfs_registry_block_storageclass_default=true
-
-openshift_storageclass_default=false
-----
-<1> It is recommended to run the integrated OpenShift Container Registry on
-nodes dedicated to "infrastructure" applications, that is applications deployed
-by administrators to provide services for the {product-title} cluster.
+openshift_storage_glusterfs_registry_block_storageclass_default=false
+----
+<1> It is recommended to run the integrated OpenShift Container Registry,
```

Comment:
Maybe 
For improved performance, run the integrated OpenShift container registry,
logging, and metrics components on nodes that are dedicated to 
infrastructure applications, the applications that administrators deploy to 
provide services for the {product-title} cluster.

Reply:
That feels clunkier than what I currently have. Forgoing the suggestion. :)

Reply:
Then at least remove the quotation marks around infrastructure. I'd also rather not have the passive "It is recommended." Style frowns on it. ;)

Reply:
I'm not sure what to do as an alternative. There is no strict requirement that the Registry, Logging, and Metrics be run on infra nodes, it IS just a recommendation. Best I can come up with is "Though not required, <apps> should be run on nodes dedicated to infrastructure applications", but that doesn't feel much better.

Reply:
Maybe we could get away with this?
For improved performance, run the integrated OpenShift container registry,
logging, and metrics components on dedicated nodes.

Reply:
Not technically correct, this is more a case of traffic/logical isolation. There is no guarantee this will improve any performance.